devel/README.md

```diff
@@ -229,10 +229,10 @@ Using Mixer as an example:
 
 ```shell
 cd $(ISTIO)/mixer
-bazel build ...
+bazel build //...
 ```
 
-This figures out what it needs to do and does not need any input from you.
+This build command figures out what it needs to do and does not need any input from you.  
```

Comment:
hmm, it doesn't work for me, here is my zsh version:

→ zsh --version
zsh 5.2 (x86_64-apple-darwin16.0)

Perhaps you are on a newer version?    Good catch on the nit. :)